release_notes/Service_Fabric_ReleaseNotes_90.md

```diff
@@ -0,0 +1,153 @@
+
+<br>
+
+## Future Retirement and Deprecation Path Callouts
+
+| Support | Option | Details |
+|-----------------|--------|---------|
+| **Retired support for ASP.NET Core 1.0** | Migrate usage to ASP.NET Core 2.1 or greater | Service Fabric ASP.NET Core packages are built against ASP.NET Core 1.0 binaries which are out of support. Starting Service Fabric 8.2, we will be building Service Fabric ASP.NET Core packages against .NET Core 3.1, .NET Core 5.0, .NET Core 6.0, .NET Framework 4.6.1. As a result, they can be used only to build services targeting .NET Core 3.1, .NET Core 5.0, .NET Core 6.0, >=.NET Framework 4.6.1 respectively.<br>For .NET Core 3.1, .NET Core 5.0, .NET Core 6.0, Service Fabric ASP.NET Core will be taking dependency on Microsoft.AspNetCore.App shared framework, whereas for NetFx target frameworks >=.NET Framework 4.6.1, Service Fabric ASP.NET Core will be taking dependency on ASP.NET Core 2.1 packages.<br>The package Microsoft.ServiceFabric.AspNetCore.WebListener will no longer be shipped, Microsoft.ServiceFabric.AspNetCore.HttpSys package should be used instead.|
+| **Deprecated support for .NET Core 2.x** | Migrate usage to .NET Core 3.1 | .NET Core runtime LTS 2.1 runtime is out of support as of Aug 21, 2021, and .NET Core runtime 2.2 is out of support from Dec 2019.<br>Service Fabric releases after those dates will drop support for Service Fabric applications running with .NET Core 2.1 & .NET Core 2.2 runtimes respectively.<br>Current applications running on .NET Core 2.x runtime will continue to work, but requests for investigations or request for changes will no longer receive support. Service Fabric .NET SDK will take a dependency on .NET runtime 3.x features to support Service Fabric .NET Core applications. |
+| **(Upcoming) Deprecation of support for .NET 5.0** | Migrate usage to .NET 6.0 or .NET Core 3.1 |.NET 5.0 runtime is reaching end of life on May 8, 2022.<br>Service Fabric releases after that date will drop support for Service Fabric applications running with .NET 5.0 runtime.<br>Current applications running on .NET 5.0 runtime will continue to work, but requests for investigations or request for changes will no longer receive support. |
+| **(Upcoming) Retirement of support for Ubuntu 16.04** | Upgrade to more recent 18.04 LTS or 20.04 LTTS | Ubuntu 16.04 LTS reached it's 5-year end-of-life window on April 30, 2021.<br>Service Fabric runtime has dropped support for 16.04 LTS and we recommend moving your clusters and applications to Ubuntu 18.04 LTS or 20.04 LTS.<br>Current applications running on it will continue to work, but requests for investigations or requests for change will no longer receive support.<br>Service Fabric runtime will soon be permenantly retiring support for Ubuntu 16.04 LTS and will no longer make packages available for it. |
```

Comment:
```suggestion
| **(Upcoming) Retirement of support for Ubuntu 16.04** | Upgrade to more recent 18.04 LTS or 20.04 LTTS | Ubuntu 16.04 LTS reached it's 5-year end-of-life window on April 30, 2021.<br>Service Fabric runtime has dropped support for 16.04 LTS, and we recommend moving your clusters and applications to Ubuntu 18.04 LTS or 20.04 LTS.<br>Current applications running on it will continue to work, but requests for investigations or requests for change will no longer receive support.<br>Service Fabric runtime will soon be permenantly retiring support for Ubuntu 16.04 LTS and will no longer make packages available for it. |
```

Reply:
^added comma